src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -77,6 +77,24 @@ namespace OpenMS
       bool operator()(const MSSpectrum& a, const MSSpectrum& b) const;
     };
 
+    /// Used to remember what subsets in a spectrum are sorted already to allow faster sorting of the spectrum
+    struct Chunk {
+      Size start;
+      Size end;
+      bool isSorted;
+      Chunk(Size a, Size b, bool c) : start(a), end(b), isSorted(c) {}
+    };
+
+    struct Chunks {
+      std::vector<Chunk> chunks_;
```

Comment:
`chunks_` are public here, but named as if private. remove the `_`

Reply:
or make it private and provide a `get` method (this is a bit cleaner because noone from outside can operate on the vector)

Reply:
made a get-Function